README.md

```diff
@@ -152,29 +152,29 @@ A tutorial using this method will soon be available in Grafana Cloud's docs.
 The following instructions will help you go from metrics to logs to traces.
 
 1. Open the TNS dashboard.
-1. Zoom in on a section of a panel with failed requests.
-1. From the panel drop-down, click Explore.
-1. In the Explore view, go to the data source drop-down and select Loki.
-1. Choose a logline with a traceID of Tempo.
+2. Zoom in on a section of a panel with failed requests.
```

Comment:
(nit) The repeated `1.`'s here will actually render as a correct numbered list in markdown. This is helpful to keep as you can easily add an item in an arbitrary position and not have to renumber everything. (But I see that other lists in this doc don't follow this convention so /shrug).